src/ReleaseHistory.md

```diff
@@ -1,5 +1,9 @@
 # SARIF Package Release History (SDK, Driver, Converters, and Multitool)
 
+## **v2.2.1** [Sdk](https://www.nuget.org/packages/Sarif.Sdk/2.2.1) | [Driver](https://www.nuget.org/packages/Sarif.Driver/2.2.1) | [Converters](https://www.nuget.org/packages/Sarif.Converters/2.2.1) | [Multitool](https://www.nuget.org/packages/Sarif.Multitool/2.2.1)
+* FEATURE: Multitool `remove` option now supports `Guids` value to remove `Result.Guid`.
+* FEATURE: Significant Baselining algorithm improvements: dynamic partialFingerprint trust, location-specific unique what property matching, 'nearby' matching, correct omitted Region property handling, correct ReportingDescriptor.DeprecatedIds handling.
```

Comment:
>partialFingerprint [](start = 66, length = 18)

Ultra-nit: Markdown backticks around code elements, as you did on the previous line.